.github/workflows/bump-jslib.yml

```diff
@@ -0,0 +1,74 @@
+name: Bump jslib
+
+on:
+  workflow_dispatch:
+
+jobs:
+  bump-jslib:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Setup git config
+        run: |
+          git config user.name = "GitHub Action Bot"
+          git config user.email = "<>"
+
+      - name: Bump jslib and commit
+        id: bump
+        run: |
+          TARGET_BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/})
+          NEW_BRANCH_NAME="auto-bump-jslib"
+
+          if [ "$TARGET_BRANCH_NAME" = "master" ]; then
+            git checkout -b $NEW_BRANCH_NAME
+            echo "[*] Checked out new branch $NEW_BRANCH_NAME"
+          fi
```

Comment:
Any issues with running this when `auto-bump-jslib` already exists in an open PR from a previous run?

Reply:
Yep - have now updated and refactored to allow for this. If there is an existing PR, it'll push to that branch instead and add a comment with the `git log` output.